iphone/TitaniumKit/TitaniumKit/Sources/API/KrollBridge.m

```diff
@@ -563,6 +563,25 @@ - (id)krollObjectForProxy:(id)proxy
   return result;
 }
 
+- (id)require:(KrollContext *)kroll path:(NSString *)path
+{
+  if (!kroll || !path) {
+    return nil;
+  }
+
+  JSContext *jsContext = [JSContext contextWithJSGlobalContextRef:kroll.context];
+  JSValue *jsResult = [jsContext.globalObject invokeMethod:@"require" withArguments:@[ path ]];
```

Comment:
This is explicitly invoking `global.require`, which probably is fine for hyperloop's usage (since I think it's always dealing with hyper loop wrappers and "absolute" require paths (relative to app root). I think the "most correct" way would be to grab the `require` function from the current context, but honestly I'm not sure it'd work (ideally we'd want the require function passed in to the current module's context/wrapper to be invoked so relative paths get resolved properly).

Reply:
Yeah, I'm not sure what else I should do here. It may be bad to assume to use `[JSContext currentContext]` here... and the caller can easily do this anyways. My goal was to resurrect this method for hyperloop which is always given an absolute path anyways.